source/game/system/map/MapdataCheckPoint.cc

```diff
@@ -0,0 +1,185 @@
+#include "MapdataCheckPoint.hh"
+#include <Common.hh>
+#include <egg/math/Vector.hh>
+#include <game/system/CourseMap.hh>
```

Comment:
`Common.hh` already included via Vector -> Stream. And vector already included in the header. And add a newline.
```suggestion
#include "MapdataCheckPoint.hh"

#include <game/system/CourseMap.hh>
```

Reply:
Updated to include quotes instead of angle brackets for game include